x-pack/plugins/fleet/server/saved_objects/migrations/security_solution/to_v8_14_0.ts

```diff
@@ -35,3 +35,28 @@ export const migratePackagePolicyToV8140: SavedObjectModelDataBackfillFn<
 
   return { attributes: updatedPackagePolicyDoc.attributes };
 };
+
+export const migratePackagePolicyEnableCapsToV8140: SavedObjectModelDataBackfillFn<
```

Comment:
it would be nice to cover with unit tests

Reply:
5fa3e37738eb9f24def912f78878866fdca4972c